CMP1903_A1_2324/Program.cs

```diff
@@ -10,10 +10,10 @@ internal class Program
     {
         static void Main(string[] args)
         {
-            /*
-             * Create a Game object and call its methods.
-             * Create a Testing object to verify the output and operation of the other classes.
-             */
+            // Create a new testing object, test the Die and test the Game
```

Comment:
With your code here you are driving the whole game via the testing class, this isn't exactly how I believe this should be done (In my opinion, I could be wrong)  I believe you should call the game code and then use the testing class call to instantiate a new game object and call it's methods there for clarity.

Although, your code as it is works fine, doing the above I would imagine would make your code cleaner.

Reply:
Yes you are correct, I have refactored the Game and Testing classes. The Testing class now has its own function to test the results of RollThreeDice()